src/main/java/edu/ucla/library/bucketeer/JobFactory.java

```diff
@@ -171,7 +171,14 @@ public Job createJob(final String aName, final File aCsvFile, final boolean aSub
                                 error.addMessage(MessageCodes.BUCKETEER_123);
                                 break;
                             } else if (fileNameIndex == columnIndex) {
-                                item.setFilePath(Optional.ofNullable(StringUtils.trimToNull(columns[columnIndex])));
+                                if (columns[columnIndex].trim().contains(" ")) {
```

Comment:
Not a big deal, but there is a Constants.SPACE constant that could have been used.